python/lsst/ts/FiberSpectrograph/csc.py

```diff
@@ -68,21 +77,20 @@ class FiberSpectrographCsc(salobj.BaseCsc):
     * 20: If there is an error taking an exposure.
     """
     def __init__(self, initial_state=salobj.State.STANDBY,
-                 initial_simulation_mode=0, index=0,
+                 initial_simulation_mode=0, index=-1,
```

Comment:
Consider requiring the index. "When in doubt refuse to guess" and we have 4 good choices.

Reply:
Done. This gave me a chance to do my first "required kwarg" thing!